frontend/src/core/utils/fluent/isPluralElement.test.js

```diff
@@ -0,0 +1,65 @@
+import isPluralElement from './isPluralElement';
+import parser from './parser';
+
+
+describe('isPluralElement', () => {
+    it('returns false for elements that are not select expressions', () => {
+        const input = 'my-entry = Hello!';
+        const message = parser.parseEntry(input);
+        const element = message.value.elements[0];
+
+        expect(isPluralElement(element)).toEqual(false);
+    });
+
+    it('returns true if all variant keys are CLDR plurals', () => {
+        const input = `
+my-entry =
+    { $num ->
+        [one] Hello!
+       *[two] World!
+    }`;
+        const message = parser.parseEntry(input);
+        const element = message.value.elements[0];
+
+        expect(isPluralElement(element)).toEqual(true);
+    });
+
+    it('returns true if all variant keys are numbers', () => {
+        const input = `
+my-entry =
+    { $num ->
+        [1] Hello!
+       *[2] World!
+    }`;
+        const message = parser.parseEntry(input);
+        const element = message.value.elements[0];
+
+        expect(isPluralElement(element)).toEqual(true);
+    });
+
+    it('returns true if one variant key is a CLDR plural and the other is a number', () => {
+        const input = `
+my-entry =
+    { $num ->
+        [one] Hello!
+       *[1] World!
+    }`;
+        const message = parser.parseEntry(input);
+        const element = message.value.elements[0];
+
+        expect(isPluralElement(element)).toEqual(true);
+    });
```

Comment:
As I stated on the actual code, `[1] foo *[other] bar` isn't actually a plural form semantically, I'd love to see a test for that.

Also, there's no test for something like `[one] foo *[male] other `, which I think would good to have here.